client/src/util/stateManager/schemaHelpers.js

```diff
@@ -90,7 +100,11 @@ export function addObsAnnoCategory(schema, name, category) {
 
 	const newSchema = _reindex(_copy(schema));
 
-	/* remove category */
-	newSchema.annotations.obsByName[name].categories.push(category);
+	/* add category, retaining presentation sort order */
+	const catAnno = newSchema.annotations.obsByName[name];
+	catAnno.categories = catLabelSort(catAnno.writable, [
+		...catAnno.categories,
```

Comment:
Spacing

Reply:
the whole file was tab indented. Fixed now.